dashboards/Valkey/Valkey_Replication.json

```diff
@@ -301,18 +325,19 @@
           "disabled": true,
           "id": "calculateField",
           "options": {
+            "alias": "",
             "binary": {
               "left": {
                 "matcher": {
                   "id": "byName",
-                  "options": "$replica_service_name"
+                  "options": "Master"
```

Comment:
Just to double check, should this Master/Slave be here or some variable should be?